XenAdmin/MainWindow.resx

```diff
@@ -2007,6 +2007,15 @@
   <data name="pluginItemsPlaceHolderToolStripMenuItem2.Text" xml:space="preserve">
     <value>PluginItemsPlaceHolder</value>
   </data>
+  <data name="manageCPUGroupToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
+    <value>277, 22</value>
+  </data>
+  <data name="manageCPUGroupToolStripMenuItem.Text" xml:space="preserve">
+    <value>Manage CPU Group...</value>

```

Comment:
I believe this needs a hotkey.

Reply:
Thanks, I change it into "Manage CPU &groups...". Is it ok?

Reply:
Yes, I don't see any other item in this menu using the G, so it's ok. G should be capital though, because in this case it is a menu item.

Reply:
Ok, I will change it. Thanks.